xarray/tests/__init__.py

```diff
@@ -325,7 +325,7 @@ def create_test_data(
             "dim1",
             pd.Categorical(
                 np.random.choice(
-                    list(string.ascii_lowercase[: np.random.randint(5)]),
+                    list(string.ascii_lowercase[: np.random.randint(1, 5)]),
```

Comment:
we have a RandomState object in this function already
```suggestion
                    list(string.ascii_lowercase[: rs.randint(1, 5)]),
```

Reply:
Good suggestion, done